### PR TITLE
Added recently-dumped daiohp2 set

### PR DESCRIPTION
**Pull request by MooglyGuy**
New working machines
--------------------
Daioh (prototype, earlier) [CoolMod, Caius, The Dumping Union]

### CODE REVIEW

**Comment by MooglyGuy** [comment]
Note: I realize that this pull request still touches the daiohp romdef in addition to the new set. Whoever added daiohp added it with capital-case hex constants rather than respecting the established lower-case hex style used throughout the entire rest of the driver.

File: src/mame/drivers/seta.cpp [hunk 1]
```diff
@@ -12324,6 +12362,7 @@ GAME( 1993, atehate,   0,        atehate,   atehate,   seta_state,     empty_ini
 GAME( 1993, daioh,     0,        daioh,     daioh,     seta_state,     empty_init,     ROT270, "Athena",                    "Daioh", 0 )
 GAME( 1993, daioha,    daioh,    daioh,     daioh,     seta_state,     empty_init,     ROT270, "Athena",                    "Daioh (earlier)", 0 )
 GAME( 1993, daiohp,    daioh,    daiohp,    daiohp,    seta_state,     empty_init,     ROT270, "Athena",                    "Daioh (prototype)", 0 )
+GAME( 1993, daiohp2,   0,        daiohp,    daiohp,    seta_state,     empty_init,     ROT270, "Athena",                    "Daioh (prototype, earlier)", 0 )
```
Comment (clawgrip) [inline src/mame/drivers/seta.cpp:12365]: Shouldn’t it be a clone of daioh?

Comment (MooglyGuy) [reply src/mame/drivers/seta.cpp:12365]: In the future, try to be more helpful with your comments. I had to go and visually compare a bunch of hashes before realizing that "gfx2" and "gfx3" were the same between daiohp2 and daiohp, which you could have made clearer in the first place.

Comment (happppp) [reply src/mame/drivers/seta.cpp:12365]: Yes it's a MAME clone set of daioh, no complicated reasons for it.

**Comment by happppp** [comment]
Nothing's wrong with the PR other than that. Thanks moogly, keep it calm.

**Comment by happppp** [comment]
Woops, I should have reworded that merge to "New working clones"

**Pull request merged**